pyvista/core/filters/uniform_grid.py

```diff
@@ -159,6 +159,81 @@ def extract_subset(self, voi, rate=(1, 1, 1), boundary=False, progress_bar=False
         fixed.copy_meta_from(result)
         return fixed
 
+    def image_dilate_erode(self, dilate_value=1, erode_value=0,
+                           kernel_size=(3, 3, 3), scalars=None,
+                           preference='points', progress_bar=False):
+        """Dilates one value and erodes another.
+
+        image_dilate_erode will dilate one value and erode another. It uses an
+        elliptical foot print, and only erodes/dilates on the boundary of the
+        two values. The filter is restricted to the X, Y, and Z axes for now.
+        It can degenerate to a 2 or 1 dimensional filter by setting the kernel
+        size to 1 for a specific axis.
+
+        Parameters
+        ----------
+        dilate_value : int or float, optional
+            The value in the dataset that will get dilated. Default: ``1``.
```

Comment:
```suggestion
            Dilate value in the dataset. Default: ``1``.
```

Reply:
I'm not certain the suggestion is strictly better here. Same for `erode`.

Reply:
But it seems I'm wrong, sorry. ITK also uses "erode value" https://itk.org/ITKExamples/src/Filtering/BinaryMathematicalMorphology/ErodeABinaryImage/Documentation.html